src/sw-build.js

```diff
@@ -0,0 +1,28 @@
+/* Here, we'll provide instructions to generate the service worker */
```

Comment:
So glad that medium article provided steps that actually work! Stoked we have the service worker set up now